.env

```diff
@@ -164,6 +162,8 @@ MF_VAULT_UNSEAL_KEY_3=
 MF_VAULT_TOKEN=
 MF_VAULT_CA_NAME=mainflux
 MF_VAULT_CA_ROLE_NAME=mainflux
+MF_VAULT_PKI_PATH=pki
```

Comment:
What are PKI_PATH and PKI_INT? Maybe add some comments, because this looks confusing. Also - are these in the README?

Reply:
@drasko [PKI](https://www.vaultproject.io/docs/secrets/pki) related vars are Vault specific. IMHO the `VAULT` prefix in var name is fine if you know the domain (and you must know it in order to use Vault).

Reply:
@drasko I added explanation fro envs in README.md, and as @nmarcetic knowledge of Vault is expected here